app/src/main/java/com/nutomic/syncthingandroid/service/Constants.java

```diff
@@ -30,6 +30,14 @@
      */
     public static final String PREF_DEBUG_FACILITIES_AVAILABLE  = "debug_facilities_available";
 
+    /**
+     * Available folder types.
+     */
+    public static final String FOLDER_TYPE_SEND_ONLY            = "readonly";
```

Comment:
I think its "sendonly"? Or atleast we were planning to rename, yet supported the old names.

Reply:
I was unsure if "sendonly" can be set already for 14.48. Let's put the new names when we have 14.49 (or future version) out and the rename took place.

Reply:
I checked how it's currently implemented upstream:
v14.48:
- Web UI writes "readonly", "readwrite" (initially, on change)
- syncthing understands "readonly", "readwrite", "sendonly", "sendreceive" in config.xml
v14.49-rc.3:
- Web UI writes "sendonly", "sendreceive" (initially, on change)
- syncthing understands "readonly", "readwrite", "sendonly", "sendreceive" in config.xml
So I would recommend to put "sendreceive" and "sendonly" when we bump syncthing to v14.49 . If we would do it now, the user can jump into a pitfall when we create a folder "sendonly" and he edits it later through the web UI to be "readwrite". Our code would - according to the constants in place - compare "readwrite" to the string "sendreceive" and assume we were "readonly" then. This can be solved by changing the constans later after 14.49 got released.

Reply:
So I'll only release the new version of the app once .49 rolls off the press.

Reply:
Ok, I prepared this PR for syncthing v0.14.49+, see commit https://github.com/syncthing/syncthing-android/pull/1170/commits/463d9cc1d3f1f139b19aa49a522cb0acce23c1c9